more_complex/variables.pkr.hcl

```diff
@@ -0,0 +1,4 @@
+variable "flavor" {
+  type = string
```

Comment:
```suggestion
  type     = string
```

Reply:
could also add a description, to make it more obvious what this variable is (and to teach best-practices)

Reply:
great idea.